sdk/cosmos/src/headers/from_headers.rs

```diff
@@ -145,146 +147,148 @@ pub(crate) fn indexing_directive_from_headers_optional(
         .to_str()?
         .parse()?)
 }
 
-pub(crate) fn lsn_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn lsn_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     Ok(headers
         .get(HEADER_LSN)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_LSN.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_LSN.to_owned()))?
         .to_str()?
         .parse()?)
 }
 
-pub(crate) fn item_lsn_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn item_lsn_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     Ok(headers
         .get(HEADER_ITEM_LSN)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_ITEM_LSN.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_ITEM_LSN.to_owned()))?
         .to_str()?
         .parse()?)
 }
 
-pub(crate) fn transport_request_id_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn transport_request_id_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     Ok(headers
         .get(HEADER_TRANSPORT_REQUEST_ID)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_TRANSPORT_REQUEST_ID.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_TRANSPORT_REQUEST_ID.to_owned()))?
         .to_str()?
         .parse()?)
 }
 
-pub(crate) fn global_committed_lsn_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn global_committed_lsn_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     let s = headers
         .get(HEADER_GLOBAL_COMMITTED_LSN)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_GLOBAL_COMMITTED_LSN.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_GLOBAL_COMMITTED_LSN.to_owned()))?
         .to_str()?;
     Ok(if s == "-1" { 0 } else { s.parse()? })
 }
 
-pub(crate) fn cosmos_llsn_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn cosmos_llsn_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     Ok(headers
         .get(HEADER_COSMOS_LLSN)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_COSMOS_LLSN.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_COSMOS_LLSN.to_owned()))?
         .to_str()?
         .parse()?)
 }
 
-pub(crate) fn cosmos_item_llsn_from_headers(headers: &HeaderMap) -> Result<u64, CosmosError> {
+pub(crate) fn cosmos_item_llsn_from_headers(headers: &HeaderMap) -> Result<u64, crate::Error> {
     Ok(headers
         .get(HEADER_COSMOS_ITEM_LLSN)
-        .ok_or_else(|| CosmosError::HeaderNotFound(HEADER_COSMOS_ITEM_LLSN.to_owned()))?
+        .ok_or_else(|| crate::Error::HeaderNotFound(HEADER_COSMOS_ITEM_LLSN.to_owned()))?
         .to_str()?
         .parse()?)
 }
 
 pub(crate) fn current_replica_set_size_from_headers(
     headers: &HeaderMap,
-) -> Result<u64, CosmosError> {
+) -> Result<u64, crate::Error> {
```

Comment:
As much as this gets repeated - and to make it easy to reference - would it make more sense to define our own `crate::Result<T>` as well? I see this in a lot of other crates.